### ISSUE TITLE
Checkout as persistent user test failing

### ISSUE THREAD

**Issue by plabadie** [SAP/spartacus#6696]
Fails on 1.5.x branch
https://jkmaster.test.c3po.b2c.ydev.hybris.com/job/spartacus-regression-tests-maintenance/31/consoleText

**Comment by Matejk00** [comment]
Tested manually and checkout path works fine, those tests seems to be flaky, sometimes they all passing, but sometimes in some cases 4 of them are failing. 

Didn't found any reason yet.

**Comment by Matejk00** [comment]
Since we decided to do manually testing let's leave this ticket for now. However we should back to it when we come up with some flakiness removal strategy.

**Comment by qjimmy** [comment]
Failing sometimes on `release/1.5.x` branch. Re-opening this ticket

**Comment by qjimmy** [comment]
PR  #6860

**Comment by qjimmy** [comment]
Removing the must label since this ticket is about the 1.5 branch and not 2.0

**Comment by marlass** [comment]
It is actually flaky on both maintenance and develop branch.

**Comment by qjimmy** [comment]
Investigated and tested on dev-7 using latest develop, sometimes fails on selecting the delivery mode during the checkout phase because of indefinite loading

![checkout-as-persistent-usere2e-s](https://user-images.githubusercontent.com/30666844/81592426-f34f0200-938b-11ea-96d7-ebc2b3781d1b.gif)

Tried to find out fix but don't know the source of error as I could not reproduce in open test runner since it seems to be only unstable in headless mode.

@marlass Could you take a look at it or mark it as flaky so jenkins skips it?

**Comment by giancorderoortiz** [comment]
Delivery method page load similar to https://github.com/SAP/spartacus/issues/8012

**Comment by giancorderoortiz** [comment]
This issue is blocked by https://cxjira.sap.com/browse/C3PO-2396

**Comment by Zeyber** [comment]
Not blocked. PRs above fix it. Cypress test was not waiting for occ call.

**Comment by giancorderoortiz** [comment]
I'm reluctant to agree given random delays in loading delivery-mode page are caused by backend, not Spartacus.
How many times did you run pr to confirm repeatability? If fix indeed works, it unblocks a lot of things...

**Comment by bgambocjaviniar** [comment]
will re-visit the PR on the azure servers

**Comment by Zeyber** [comment]
Note: Partially fixed by #8101 but there is another issue with the test locking here
![image](https://user-images.githubusercontent.com/11735817/91042088-7fbac300-e611-11ea-8b3b-3b6b9bd24f46.png)

**Comment by psmul** [comment]
Ran both `checkout-as-persistent-user.e2e-spec.ts` and `checkout-as-persistent-user-mobile.e2e-spec.ts` more than 10 times and no error occurred. Closing this one. If error re-appears, fresh ticket will get created.
